src/hotspot/share/runtime/flags/jvmFlagConstraintsRuntime.cpp

```diff
@@ -53,10 +53,10 @@ JVMFlag::Error ObjectAlignmentInBytesConstraintFunc(int value, bool verbose) {
 
 // Need to enforce the padding not to break the existing field alignments.
 // It is sufficient to check against the largest type size.
-JVMFlag::Error ContendedPaddingWidthConstraintFunc(intx value, bool verbose) {
+JVMFlag::Error ContendedPaddingWidthConstraintFunc(int value, bool verbose) {
   if ((value % BytesPerLong) != 0) {
     JVMFlag::printError(verbose,
-                        "ContendedPaddingWidth (" INTX_FORMAT ") must be "
+                        "ContendedPaddingWidth (%u) must be "
```

Comment:
Why %u when it is an int?

Reply:
I had a temporary version where it was uint that didn't do the range check message correctly.  Fixed.